src/server.ls

```diff
@@ -13,7 +13,8 @@ module.exports =
 
   before-all: (done) ->
     mongo-db-name = "exosphere-mongo-service-#{env}"
-    MongoClient.connect "mongodb://localhost:27017/#{mongo-db-name}", N (mongo-db) ->
+    mongo-address = if env is \test then \localhost else \mongo
```

Comment:
Yes we can, I have included that now as part of `bin/spec`, what do you think of it?